### PR TITLE
add some test case for ges

### PR DESCRIPTION
**Pull request by wean2016**
### Description section
1. Update the misspelling 'deafult' to 'default' in TestPC.py
2. Unify the initial variable names of the GES and PC algorithms
3. Upload some test data and graph files for GES
4. Added some unit tests for GES

### Test plan section
python -m unittest TestGES.py # should pass
python -m unittest TestPC.py # should pass`
testpc
TestGES

### CODE REVIEW

File: tests/TestGES.py [hunk 1]
```diff
@@ -1,188 +1,123 @@
...
-import matplotlib.pyplot as plt
 import numpy as np
-
 from causallearn.search.ScoreBased.GES import ges
 
 
+######################################### Test Notes ###########################################
+# All the benchmark results of loaded files (e.g. "./TestData/benchmark_returned_results/")    #
+# are obtained from the code of causal-learn as of commit                                      #
+# https://github.com/cmu-phil/causal-learn/commit/f211557a (07-08-2022).                       #
```
[the remark this answers is not in the document]
Comment (tofuwen) [reply tests/TestGES.py:15]: But using it is also probably fine --- I checked the commit history and all users' commits are there if merged. 

Just to make sure, when generating the data, you used the code AFTER you made the changes or before the changes?

[the remark this answers is not in the document]
Comment (wean2016) [reply tests/TestGES.py:15]: I generated the data before the changes. In addition, the changes have not substantially modified the logic of the GES.py algorithm. Actually, these commits only modified the initial variables name of GES.py.

By the way, I wonder should I firstly close this pull request and then propose a new pull request which contains a test plan section?

[the remark this answers is not in the document]
Comment (tofuwen) [reply tests/TestGES.py:15]: Cool, then directly use b51d788 as your commit id will work. Just change the commit id, and no more changes are needed.

You don't need to close this pull request to edit your test plan section. Just edit your first message in "Conversation" to include:
1) description section
2) test plan section

[the remark this answers is not in the document]
Comment (wean2016) [reply tests/TestGES.py]: I have updated the related code file and pr message. Please check this pr at your convenience.

**Comment by tofuwen** [comment]
Thanks so much for the great work!

I think the code is alrady good enough --- I just need to look at the result to make sure the current code output makes sense.

1. As I said before, please add "a screenshot of your terminal after you run the command to test". I'd like to see the result and print message before accepting this PR. It would be great next time, you can add the test plan section in your initial PR. Test plan is the most important section for every PR. 

2. A very small nit: please make "Description section" and "Test plan section" a header instead of the current paragraph. 

It should looks like this (click the "H" in your box):
Screen Shot 2022-07-11 at 17 00 15


### Description

### Test Plan

**Comment by tofuwen** [comment]
Great job, thanks so much!

I think this PR is ready to be pushed. @kunwuz

**Review by tofuwen — APPROVED** [review]

**Pull request merged**